### PR TITLE
provide geocoordinates for projects

### PR DESCRIPTION
**Pull request by infabo**

### CODE REVIEW

File: src/Justimmo/Model/Project.php [hunk 1]
```diff
@@ -240,6 +240,52 @@ class Project
...
+        return $this->latitude;
+    }
+
+    /**
+     * @param float $latitude
+     * @return Project
+     */
+    public function setLatitude($latitude)
+    {
+        $this->latitude = $latitude;
```
Comment (19Gerhard85) [inline src/Justimmo/Model/Project.php:267]: Please add a new line afterwards.

File: src/Justimmo/Model/Project.php [hunk 2]
```diff
@@ -240,6 +240,52 @@ class Project
...
+        return $this->longitude;
+    }
+
+    /**
+     * @param float $longitude
+     * @return Project
+     */
+    public function setLongitude($longitude)
+    {
+        $this->longitude = $longitude;
```
Comment (19Gerhard85) [inline src/Justimmo/Model/Project.php:286]: Please add a new line afterwards.

File: src/Justimmo/Model/Wrapper/V1/ProjectWrapper.php [hunk 3]
```diff
@@ -141,6 +141,12 @@ public function transformSingle($data)
             }
         }
 
+        if (isset($xml->geokoordinaten)) {
+            $coord = $this->attributesToArray($xml->geokoordinaten->attributes());
+            $project->setLatitude((float) $coord['breitengrad']);
```
Comment (19Gerhard85) [inline src/Justimmo/Model/Wrapper/V1/ProjectWrapper.php]: Please cast to (double)

File: src/Justimmo/Model/Wrapper/V1/ProjectWrapper.php [hunk 4]
```diff
@@ -141,6 +141,12 @@ public function transformSingle($data)
...
+            $project->setLongitude((float) $coord['laengengrad']);
```
Comment (19Gerhard85) [inline src/Justimmo/Model/Wrapper/V1/ProjectWrapper.php]: Please cast to (double)

**Review by 19Gerhard85 — CHANGES_REQUESTED** [review]

**Comment by infabo** [comment]
resolved requested changes.

**Review by 19Gerhard85 — APPROVED** [review]

**Pull request merged**